src/AppBundle/Entity/Contributor.php

```diff
@@ -151,15 +151,15 @@ private function markUpdated(): Contributor
     /**
      * Get id.
      */
-    public function getId(): int
+    public function getId(): ?int
```

Comment:
I'm not sure I understand these new type hints on getter and setter... The properties type hints and ORM declaration didn't change, why only the getter/setter types should be changed ?

Reply:
Because when EasyAdmin creates an empty form, it does so by instanciating an empty `Contributor`, and then access all the fields to populate the creation form view.

I added here and there those type-hints, but sometimes got to restrictive given this operational principle.

Reply:
That's really ***ed up! Is there any way to use Proxy/Model/Anyhting instead of the entity to do so? 
Might be an issue with the the handling of the type hinting and not the fact that easyadmin create a new empty instance, might be a thing with the form component and the types instead...
I think it's important to give it some thought because otherwise we'll simply screw over the domain, and it wont mean anything anymore.

Reply:
Yes, I don’t like it neither :-(

I suggest we create a separate issue to see what we can do about this …

Reply:
OK